app/assets/stylesheets/config/typography.css.scss

```diff
@@ -4,7 +4,6 @@ body {
   color: black;
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
-  font-smoothing: antialiased;
```

Comment:
there's no such property (there's [`font-smooth`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth) but it's non standard and I wouldn't use it.)

btw there are reason's not to use smoothing, it's not always the best idea, e.g. contrast is worse
- http://usabilitypost.com/2012/11/05/stop-fixing-font-smoothing/
- https://www.zachleat.com/web/font-smooth/

Reply:
Cool stuff! Thanks for sharing